rs/embedders/src/wasmtime_embedder.rs

```diff
@@ -905,22 +908,9 @@ impl WasmtimeInstance {
                 "Memory tracking disabled. Returning empty list of dirty pages"
             );
             Ok(PageAccessResults {
-                wasm_dirty_pages: vec![],
-                wasm_num_accessed_pages: 0,
-                wasm_read_before_write_count: 0,
-                wasm_direct_write_count: 0,
-                wasm_sigsegv_count: 0,
-                wasm_mmap_count: 0,
-                wasm_mprotect_count: 0,
-                wasm_copy_page_count: 0,
                 stable_dirty_pages,
-                stable_accessed_pages: 0,
-                stable_read_before_write_count: 0,
-                stable_direct_write_count: 0,
-                stable_sigsegv_count: 0,
-                stable_mmap_count: 0,
-                stable_mprotect_count: 0,
-                stable_copy_page_count: 0,
+                stable_accessed_pages,
```

Comment:
Drive-by fix: `stable_accessed_pages` were not reported when there was no heap memory (likely a rare use case).